### ISSUE TITLE
after sync reset, devices are not visible anymore if you try to re-sync

### ISSUE THREAD

**Issue by cezaraugusto** [brave/browser-laptop#13258]
STR:

1. `npm run start` and `npm run start2`
2. Sync two devices
3. Ensure they show up in the devices table
4. Hit "reset sync"
5. Repeat steps to sync two devices
6. No device is shown in the device table

Expected result: 

Reset sync should not affect devices display.

This is a blocking issue after #13197 lands, since in Sync v2 there's no way to complete setup if no device is shown

**Comment by bsclifton** [comment]
Closing as `wontfix` since we're going to switch gears and focus more on Brave Core. This is captured there with https://github.com/brave/brave-browser/issues/356
